### PR TITLE
8286069: keytool prints out wrong key algorithm for -importpass command

### PR DESCRIPTION
**Pull request by wangweij**
Since `keytool -importpass` always uses `KeyFactory.getInstance("PBE")` to generate the secret key, and "PBE" is an alias of "PBEwithMD5andDES" inside the SunJCE security provider, its `getAlgorithm` is always `PBEwithMD5andDES`.

This code change modifies it to "PBE".

Note that I haven't chosen the `-keyalg` option value here because it is actually the algorithm used to protect the PBE secret key entry. It's a cipher algorithm instead of a key algorithm.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (1 reviews required, with at least 1 reviewer)

### Issue
 * [JDK-8286069](https://bugs.openjdk.java.net/browse/JDK-8286069): keytool prints out wrong key algorithm for -importpass command

### Reviewers
 * [Hai-May Chao](https://openjdk.java.net/census#hchao) (@haimaychao - Committer) ⚠️ Review applies to [e99bdc32](https://git.openjdk.java.net/jdk/pull/8520/files/e99bdc327d297b97357b787c928bbeb847a90956)
 * [Valerie Peng](https://openjdk.java.net/census#valeriep) (@valeriepeng - **Reviewer**) ⚠️ Review applies to [e99bdc32](https://git.openjdk.java.net/jdk/pull/8520/files/e99bdc327d297b97357b787c928bbeb847a90956)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8520/head:pull/8520` \
`$ git checkout pull/8520`

Update a local copy of the PR: \
`$ git checkout pull/8520` \
`$ git pull https://git.openjdk.java.net/jdk pull/8520/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8520`

View PR using the GUI difftool: \
`$ git pr show -t 8520`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8520.diff">https://git.openjdk.java.net/jdk/pull/8520.diff</a>

</details>

### CODE REVIEW

**Review by haimaychao — APPROVED** [review]
LGTM.

File: test/jdk/sun/security/pkcs12/ImportPassKeyAlg.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        if (algorithm != null) {
+            cmd += " -keyalg " + algorithm;
+        }
+
+        SecurityTools.setResponse("changeit\nchangeit\n");
+        SecurityTools.keytool(cmd)
+                .shouldHaveExitValue(0)
+                .shouldContain("Generated PBE secret key");
+
+        // The aid of a protected entry (at 110c010c01010c0 inside p12) is:
```
Comment (valeriepeng) [inline test/jdk/sun/security/pkcs12/ImportPassKeyAlg.java:75]: nit: use "algorithm id" instead.

**Review by valeriepeng — APPROVED** [review]

File: test/jdk/sun/security/pkcs12/ImportPassKeyAlg.java (hunk 1 above)
Comment (wangweij) [reply test/jdk/sun/security/pkcs12/ImportPassKeyAlg.java:75]: No problem.

**Pull request closed without merging**